assignments/arrays.js

```diff
@@ -63,32 +65,50 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a ` + inventory[32]['car_year'] + ` ` + inventory[32].car_make  +  " " + inventory[32]['car_model'] );
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length - 1].car_make + " " + inventory[inventory.length - 1].car_model;
```

Comment:
Since last car was already declared - you can simply state 'lastCar = etc.'.
Try pulling complex lines out of your code and referencing them with variables, like so:
let finalCar = inventory.length - 1;
lastCar = inventory[finalCar].car_make + " " + inventory[finalCar].car_model;

It creates DRY and semantic code.